extension/pgtelemetry--1.3.sql

```diff
@@ -0,0 +1,512 @@
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+--\echo Use "CREATE EXTENSION pgtelemetry" to load this file. \quit
+
+-- filter non-backend connections from pg_stat_activity
+create view client_stat_activity as select * from pg_stat_activity where backend_type = 'client backend';
+
+-- disk space
+CREATE VIEW relation_total_size AS
```

Comment:
Inconsistent formatting. I suggest lowercasing all sql commands and identifiers (rationale: SQL is a programming language, you don't write programs in uppercase).

Reply:
😱 lowercase sql commands what happened to the world?

Reply:
I guess the world has moved toward the editors with syntax highlighting for the SQL reserved words, so you don't have to waste extra efforts capitalizing them.

Reply:
but postgres is capitalizing them already (e.g in pg_dump)
and for the postgres project itself the majority 3/4 seems to use uppercase commands as well

```
➜  postgresql git:(REL_10_STABLE) grep -r "select " . | wc -l
   12202
➜  postgresql git:(REL_10_STABLE) grep -r "SELECT " . | wc -l
   38286
➜  postgresql git:(REL_10_STABLE) grep -r "create table" . | wc -l
    1316
➜  postgresql git:(REL_10_STABLE) grep -r "CREATE TABLE" . | wc -l
    3296
```

Reply:
Good I wrote "suggest" instead of "insist" :-)  The whole discussion can become bike-shedding if are not careful. IMO we should adopt one style and be consistent. My preference would be lowercase, for the reasons described above (it's easier to type and not less readable).

Reply:
Why pg_dump is doing this is actually less clear to me, as dumps are normally not designed for consumption by humans and should not matter much which capitalization is used there. Perhaps it's just that "we always did it this way" argument...